### ISSUE TITLE
sebs protocol issue with Safari 13.1 on Mac 

### ISSUE THREAD

**Issue by anbrusi** [SafeExamBrowser/seb-mac#56]
In our LMS I build and store exam start files server side. Exams are started by clicking on a link with the sebs protocol like:

"sebs://istest2.ch/sebfiles/u_162_DT_15581_t_1590315260.seb"

This works fine with Chrome, Firefox and Edge, but not with Safari 13.1. The first click starts the local seb file. Subsequent clicks on the same link start the correct file stored on the server. After a certain amount of time a click on the same link reverts to the local file and everything starts over again.

Is this a known issue or did I miss something in the configuration file?
I suspect it is more a Safari issue then a SEB issue, but I am grateful for help.

Environment: Mac OSX 10.15.4, SEB 2.1.4, Safari 13.1

**Comment by danschlet** [comment]
Thanks for your report. I have to investigate this issue further. In my case it happened also with older versions of Safari, but only on some of my (developer) machines. On other Macs, it works perfectly.

After some initial research I think a Safari or macOS system database containing (custom) URL protocols sometimes gets temporarily corrupted for the sebs URL protocol, so that it sometimes works, sometimes not. I had the impression that it doesn't work after waking up the Mac from sleep. I think I didn't try other browsers, I will try to replicate if it really only happens from Safari.

**Comment by danschlet** [comment]
Needs to be fixed in Moodle, file an issue at moodle.org

**Comment by anbrusi** [comment]
I do not agree with Your conclusion. It is not a Moodle issue. I discovered it while programming SEB support for www.istest2.ch. May be it is a Safari issue.

I attach a *.seb file generated by www.istest2.ch (Exit password "exit"). If the file is started by double clicking on it on the local machine, it works ok. If the link to the to the file on a server is clicked in Safari, SEB starts in a local configuration. If SEB is terminated and the link is clicked again SEB starts in the configuration given by the file. The issue persists in Safari 14.0.3 but does not araise when the link is clicked in Chrome or Firefox.
I suspect the problem has something to do with the protocol by which *.seb files are started.

I had to rename the *.seb file to *.zip, to be able to upload  #it

[u_162_DT_33574_t_1616841508 Kopie.zip](https://github.com/SafeExamBrowser/seb-mac/files/6215549/u_162_DT_33574_t_1616841508.Kopie.zip)

<img width="1256" alt="Bildschirmfoto 2021-03-27 um 11 42 21" src="https://user-images.githubusercontent.com/36796791/112718776-6b1b1200-8ef5-11eb-8db4-702b18622529.png">
